service/discovery/azure/compute.go

```diff
@@ -344,22 +349,43 @@ func (d *azureComputeDiscovery) handleWebApp(webApp *armappservice.Site, config
 			TransportEncryption: getTransportEncryption(webApp.Properties, config),
 		},
 		PublicAccess: publicNetworkAccess,
-		Redundancy:   getRedundancy(webApp),
+		Redundancy:   d.getRedundancy(webApp),
 	}
 }
 
-func getRedundancy(app *armappservice.Site) *voc.Redundancy {
-	r := &voc.Redundancy{}
-	switch util.Deref(app.Properties.RedundancyMode) {
-	case armappservice.RedundancyModeNone:
-		break
-	case armappservice.RedundancyModeActiveActive:
-		r.Zone = true
-	case armappservice.RedundancyModeFailover, armappservice.RedundancyModeGeoRedundant:
-		r.Zone = true
-		r.Geo = true
+// getRedundancy returns for a given web app/function the redundancy in the voc format
+func (d *azureComputeDiscovery) getRedundancy(app *armappservice.Site) (r *voc.Redundancy) {
```

Comment:
Why do you get the information from the PlansClient and no longer from the site object? And why is the Geo property not longer set?

Reply:
Because the version before was hacky and not correct. The information about zone redundancy is stored in the respective service plan. And there, only the Zone Property is set, no information about Geo-Redundancy. We also only want Zone Redandancy for now, so that is enough.